CONTRIBUTING.md

```diff
@@ -24,6 +24,18 @@ serving the tests of Nmstate.
 
 - `./examples/` - Contains YAML examples for different configurations.
 
+- `./rust/src/lib/ifaces/` Contains the classes including the attributes of the 
+interfaces.
+ 
+- `./rust/src/lib/nispor/` Contains the nmstate nispor plugin, currently for 
+querying kernel network status. 
+
+- `./rust/src/lib/nm/` Contains the code related to NetworkManager communication and structures.
+
+- `./rust/src/lib/ovsdb/` Contains the code related to ovsdb communication and structures. 
+
+- `./rust/src/cli/` - Contains command lines tools.
```

Comment:
There are other subdirs under the `./rust/src/lib` like `policy`, `query_apply`, `statistic` etc. It might be good to also document them here.